bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -720,14 +720,16 @@ public String registerConnectionAsApp(String oAutRedirectUrl)
         if (StringUtils.isEmpty(this.refreshToken)) {
             throw new ConnectionException("Error: No refresh token received");
         }
-        String usersMeResponseJson = makeRequestAndReturnString("GET",
-                "https://alexa.amazon.com/api/users/me?platform=ios&version=2.2.223830.0", null, false, null);
-        JsonUsersMeResponse usersMeResponse = parseJson(usersMeResponseJson, JsonUsersMeResponse.class);
-
-        URI uri = new URI(usersMeResponse.marketPlaceDomainName);
-        String host = uri.getHost();
-        setAmazonSite(host);
         try {
+            exhangeToken();
```

Comment:
Is this a typo in the (existing) method name? Shouldn't it be called `exchangeToken`?

Reply:
Yes, you are right. It is an existing type. I will fix it. 
@kaikreuzer btw, when do you expect the release for version 2.5?

Reply:
After a few more milestones, but still this year.